### PR TITLE
Feature/logout

### PR DESCRIPTION
**Pull request by marouane97**

### CODE REVIEW

File: dova/www/templates/dashboard.html [hunk 1]
```diff
@@ -1,8 +1,43 @@
 <ion-view view-title="Dashboard" hide-back-button="true" id="page10" ng-init="loadEnrollments()" cache-view="false">
-  <ion-content padding="true" class="has-header">
-    <ion-list id="dashboard-list9" ng-repeat="enrollment in enrollments">
+  <ion-content padding="true" class="has-header manual-remove-top-padding" scroll="false">
 
-      <ion-item class="item-thumbnail-left item-icon-right  course" id="dashboard-list-item45" ng-click="toAssignments(enrollment.id)" nav-direction="forward">
+    <ion-scroll class="top-container">
```
Comment (NealSun96) [inline dova/www/templates/dashboard.html]: What are those? The new dashboard design?

Comment (marouane97) [inline dova/www/templates/dashboard.html]: using an ion-scroll allows me to put the content of the page all in that container, and have it be scrollable, so I can add a form to the bottom of the page, aka the delete button.

**Comment by NealSun96** [comment]
@marouane97 The red button is ugly af. 
Sorry, but it really is. Black and white maybe?

**Comment by marouane97** [comment]
it's supposed to stand out, do u want it to be blue or green?

**Comment by NealSun96** [comment]
@marouane97 I would disagree on it being standout, cuz logging is something students barely do. It should fit in the theme

**Comment by marouane97** [comment]
i changed it to blue.

**Comment by NealSun96** [comment]
Doable I guess. Let's merge

**Pull request merged**